.github/workflows/release.main.kts

```diff
@@ -38,98 +30,48 @@ val SIGNING_PASSWORD by Contexts.secrets
-                ),
-            )
-            run(
-                name = "Publish",
-                command = "./gradlew publishAllPublicationsToSonatypeReleaseRepository --stacktrace",
-                env =
-                mapOf(
-                    "ORG_GRADLE_PROJECT_snake-kmp.ossrhUsername" to expr { SONATYPE_USERNAME },
-                    "ORG_GRADLE_PROJECT_snake-kmp.ossrhPassword" to expr { SONATYPE_PASSWORD },
-                    "ORG_GRADLE_PROJECT_snake-kmp.ossrhStagingRepositoryId" to expr { stagingRepoJob.outputs.repositoryId },
-                    "ORG_GRADLE_PROJECT_snake-kmp.signing.keyId" to expr { SIGNING_KEY_ID },
-                    "ORG_GRADLE_PROJECT_snake-kmp.signing.key" to expr { SIGNING_KEY },
-                    "ORG_GRADLE_PROJECT_snake-kmp.signing.password" to expr { SIGNING_PASSWORD },
-                )
-            )
-        }
-
     job(
-        id = "close-staging-repo",
-        runsOn = RunnerType.UbuntuLatest,
-        condition = expr { publishJob.result.eq(AbstractResult.Status.Success) },
-        needs = listOf(stagingRepoJob, publishJob),
+        id = "release",
+        runsOn = RunnerType.MacOSLatest,
     ) {
+        uses(action = Checkout())
         uses(
-            action = ReleaseNexusStagingRepo(
-                username = expr { SONATYPE_USERNAME },
-                password = expr { SONATYPE_PASSWORD },
-                stagingRepositoryId = expr { stagingRepoJob.outputs.repositoryId },
-            )
+            name = "Set up Gradle Daemon JDK",
+            action = SetupJava(
+                javaVersion = "21",
+                distribution = Temurin,
+            ),
         )
-    }
-    job(
-        id = "drop-staging-repo",
-        runsOn = RunnerType.UbuntuLatest,
-        condition = expr { "!${cancelled()} && ${publishJob.result.neq(AbstractResult.Status.Success)}" },
-        needs = listOf(stagingRepoJob, publishJob),
-    ) {
         uses(
-            action = DropNexusStagingRepo(
-                username = expr { SONATYPE_USERNAME },
-                password = expr { SONATYPE_PASSWORD },
-                stagingRepositoryId = expr { stagingRepoJob.outputs.repositoryId },
+            name = "Cache Kotlin Konan",
+            action = Cache(
+                path = listOf(
+                    "~/.konan/**/*",
+                ),
+                key = "kotlin-konan-${expr { runner.os }}",
+            ),
+        )
+        uses(
+            name = "Set up Gradle",
+            action = ActionsSetupGradle(
+                gradleVersion = "wrapper",
+            ),
+        )
+        run(
+            name = "Publish",
+            command = "./gradlew publishToMavenCentral --no-configuration-cache --stacktrace",
+            env =
+            mapOf(
+                "ORG_GRADLE_PROJECT_mavenCentralUsername" to expr { SONATYPE_USERNAME },
+                "ORG_GRADLE_PROJECT_mavenCentralPassword" to expr { SONATYPE_PASSWORD },
```

Comment:
question: Have you updated the username and password? For publishing to Sonatype Central you need to obtain a new pair of username and password

https://central.sonatype.com/account -> Generate User Token